src/index.js

```diff
@@ -0,0 +1,146 @@
+const state = {
+  temp: 50,
+  city: 'Bloomington',
+  lat: 39.1670396,
+  lon: -86.5342881,
+  sky: 'sunny',
+};
+
+const upButton = document.getElementById('up-arrow');
+const downButton = document.getElementById('down-arrow');
+const tempText = document.getElementById('current-temp');
+const ground = document.getElementById('ground');
+const cityHeader = document.getElementById('city-name');
+const cityInput = document.getElementById('city');
+const realTemp = document.getElementById('realtime-temp');
+const skySelect = document.getElementById('sky-select');
+const skyLine = document.getElementById('sky');
+const landscape = document.getElementById('landscape');
+const cityReset = document.getElementById('city-reset');
+
+const setTempColor = (temp) => {
+  if (temp >= 80) {
+    tempText.style.color = 'red';
+  } else if (temp >= 70) {
+    tempText.style.color = 'orange';
+  } else if (temp >= 60) {
+    tempText.style.color = 'yellow';
+  } else if (temp >= 50) {
+    tempText.style.color = 'green';
+  } else {
+    tempText.style.color = 'teal';
+  }
+};
+
+const setGroundLayout = (temp) => {
+  if (temp >= 80) {
+    ground.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (temp >= 70) {
+    ground.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (temp >= 60) {
+    ground.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else {
+    ground.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
```

Comment:
A further refactor you could make is to combine `setTempColor` and `setGroundLayout` especially since the temp values you used to switch between different colors and ground layouts are the same. 

You could do something like `setColorAndLayout()`:
```javascript
if (temp >= 80) {
    ground.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
    tempText.style.color = 'red';
  } 
```

If you choose to iterate through an object with like I suggested above, then your object could look like:
```javascript
const COLORS_AND_BACKGROUND = [ 
  { upperBound: 49, color: "teal", ground: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"}
]